ArduCopter/mode_guided.cpp

```diff
@@ -588,32 +588,32 @@ void ModeGuided::set_vel_accel_NEU_m(const Vector3f& vel_neu_ms, const Vector3f&
     set_yaw_state_rad(use_yaw, yaw_rad, use_yaw_rate, yaw_rate_rads, relative_yaw);
 
     // set velocity and acceleration targets and zero position
-    guided_pos_target_neu_m.zero();
+    guided_pos_target_ned_m.zero();
     guided_is_terrain_alt = false;
-    guided_vel_target_neu_ms = vel_neu_ms;
-    guided_accel_target_neu_mss = accel_neu_mss;
+    guided_vel_target_ned_ms = vel_ned_ms;
+    guided_accel_target_ned_mss = accel_ned_mss;
     update_time_ms = millis();
 
 #if HAL_LOGGING_ENABLED
     // log target
     if (log_request) {
-        copter.Log_Write_Guided_Position_Target(guided_mode, guided_pos_target_neu_m, guided_is_terrain_alt, guided_vel_target_neu_ms, guided_accel_target_neu_mss);
+        copter.Log_Write_Guided_Position_Target(guided_mode, guided_pos_target_ned_m, guided_is_terrain_alt, guided_vel_target_ned_ms, guided_accel_target_ned_mss);
     }
 #endif
 }
 
-// set_pos_vel_NEU_m - set guided mode position and velocity target
-bool ModeGuided::set_pos_vel_NEU_m(const Vector3p& pos_neu_m, const Vector3f& vel_neu_ms, bool use_yaw, float yaw_rad, bool use_yaw_rate, float yaw_rate_rads, bool relative_yaw)
+// set_pos_vel_NED_m - set guided mode position and velocity target
```

Comment:
not a blocker but I'd be happy removing the redundant method name from the beginning of the comment